code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -139,6 +139,16 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	siemens_coefficient = 2
 
+/obj/item/clothing/suit/bomj
+	name = "bomj coat"
+	desc = "A coat padded with synthetic insulation."
+	icon_state = "bomj"
+	item_state = "bomj"
```

Comment:
It needs at least some armor protection. On a lever of other coats.

Reply:
Any specific values?

Reply:
Literally just copypaste some coat.